lib/fluent/plugin/out_logdna.rb

```diff
@@ -58,11 +60,12 @@ def format(tag, time, record)
     def write(chunk)
       body = chunk_to_body(chunk)
       response = send_request(body)
-      raise 'Encountered server error' if response.code >= 400
+      raise "Encountered server error" if response.code >= 400
+
       response.flush
     end
 
-    private
+      private
```

Comment:
nit: private keyword should not be indented like this

Reply:
well, @mdeltito, it got corrected by `RuboCop` and I reverted it back but not sure whether it will fail or not

Reply:
if thats what the lint rules you have here prefer, thats fine with me - I haven't seen that convention anywhere, but I also don't have strong opinions on it (other than "we should have linting" 😄 )